fluentd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM fluent/fluentd:v1.1.3-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+EXPOSE 24284/udp
```

Comment:
Where do you use this port?

Reply:
Ha good catch. This is typo that somehow slipped here.